pom.xml

```diff
@@ -94,6 +94,12 @@
       <version>1.2.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
```

Comment:
Why are you adding `commons-codec`?

Reply:
Without it I get:
```
[ERROR] COMPILATION ERROR : 
[ERROR] maven-git-code-format/src/test/java/com/cosium/code/format/AbstractTest.java:[16,39] package org.apache.commons.codec.digest does not exist
```

jgit 4.9 was pulling it in transitively and 5.4 no longer does

Reply:
Ok make sense